### ISSUE TITLE
[M0-Dev] ¿Qué es un libro?

### ISSUE THREAD

**Issue by JoseCarlosPPK** [fjgallardo00/Akinah#9]
Buenas @fjgallardo00. Entiendo que de un libro se puedan saber los géneros literarios a los que pertenece y el título de este. ¿Es necesario conocer el autor de los libros? ¿Es necesario tener algún identificador del libro o el propio título basta? ¿Debe recoger algún otro dato?

**Comment by JoseCarlosPPK** [comment]
Por ejemplo, un libro puede recoger la puntuación que le han dado los usuarios, el ISBN como identificador, el idioma en los que se puede leer... ¿Esos datos son necesarios?

**Comment by fjgallardo00** [comment]
Sí, es necesario conocer el autor, pues una persona puede que esté buscando libros que pertenezcan a un autor concreto.  También, los libros tendrán un identificador para poder diferenciarlos entre ellos. Y habría que añadir también más datos:
- Editorial
- Puntuación
- Idioma
- Formato
- Nº páginas
- Imagen portada
- Puntuación (rating)
- Idioma
- ISBN
